kolibri/core/auth/utils.py

```diff
@@ -20,3 +26,134 @@ def create_adhoc_group_for_learners(classroom, learners):
     for learner in learners:
         Membership.objects.create(user=learner, collection=adhoc_group)
     return adhoc_group
+
+
+models_to_copy = [
+    ContentSessionLog,
+    ContentSummaryLog,
+    MasteryLog,
+    AttemptLog,
+    UserSessionLog,
+]
+
+user_fields = [
+    "gender",
+    "birth_year",
+    "id_number",
+]
+
+log_fields = {
```

Comment:
Given how hard it can be to remember to add stuff to things like this when models change, I wonder whether a blocklist might be safer than an allowlist?

Or, we might even consider having both! Then the tests could assert that there are no fields on the models that have not been listed in either the allowlist or the blocklist, forcing someone adding a new field to determine which one it belongs in?